README.md

```diff
@@ -89,18 +97,10 @@ We leverage these qualities to produce what we call ***strata***, a
 feature unique to the condo model. See [Condo Strata](#condo-strata) for
 more information about how strata is used and calculated.
 
-> :warning: **NOTE** :warning:
->
-> Recently, the CCAO has started to manually collect high-level
-> condominium data, including total building square footage and
-> estimated unit square footage/number of bedrooms. This data is sourced
-> from listings and a number of additional third-party sources and is
-> available for the North and South triads only.
```

Comment:
**issue:** Now available for the city, yeah?

Reply:
Yep. That's part of why I removed part of this language and shifted the rest to line 73.